bitmap.go

```diff
@@ -203,16 +205,35 @@ func resize(capacity, v int) int {
 
 // MarshalJSON returns encoded string representation for the bitmap
 func (dst Bitmap) MarshalJSON() ([]byte, error) {
-	return json.Marshal(dst.ToBytes())
+	size := len(dst)
+	vals := make([]string, size)
+
+	for i, x := range dst {
+		// convert each uint64 into 16 * 4-bit hexadecimal character
+		vals[size-i-1] = fmt.Sprintf("%.16X", x)
```

Comment:
Is sprintf efficient for this?

Reply:
Was thinking of keeping it simple, since Marhsalling/Unmarshalling isn't done that frequently. Will update it.